.github/workflows/dapr-bot.yml

```diff
@@ -18,6 +18,7 @@ jobs:
             // TODO: Read owners from OWNERS file.
             const owners = [
               'artursouza',
+              'mcandeia',
```

Comment:
Since it is already sorted by the alphabet, do you want to keep doing that? 😄

Reply:
ops, fixed!